src/lib/evil/evil_api.h

```diff
@@ -1,25 +1,25 @@
 #ifndef EVIL_API_H
 #define EVIL_API_H
 
-#ifdef EAPI
-#undef EAPI
+#ifdef EVIL_API
+#undef EVIL_API
```

Comment:
If `EVIL_API` is already defined shouldn't it yield an error? As it shouldn't be already defined?